slp_tf/slp_tf/tfplan/transformers/tfplan_dataflow_creator.py

```diff
@@ -1,13 +1,135 @@
+import uuid
+
 from networkx import DiGraph
+from typing import Dict, List
 
-from otm.otm.entity.otm import OTM
+from otm.otm.entity.dataflow import Dataflow
 from slp_tf.slp_tf.tfplan.transformers.tfplan_transformer import TfplanTransformer
+from slp_tf.slp_tf.tfplan.tfplan_objects import TfplanOTM, TfplanComponent, TfplanSecurityGroup
+from slp_tf.slp_tf.tfplan.graph.relationships_extractor import RelationshipsExtractor
+
+
+def have_components_relationship(first: TfplanComponent, second: TfplanComponent):
```

Comment:
Similar to the function __is_parent i created in the singleton:
 maybe an utility is needed for set the values of :
 Relationship: children, parent, ancestor, or children of a children